e2e/push/push.go

```diff
@@ -39,7 +40,7 @@ func (c *ctx) testPushCmd(t *testing.T) {
 		desc          string // case description
 		dstURI        string // destination URI for image
 		imagePath     string // src image path
-		expectSuccess bool   // singularity should exit with code 0
+		expectSuccess bool   // expectation regarding the test's success
```

Comment:
In other tests I've seen this field changed to something like`expectExitCode` and than using that value instead of doing further checking around the `boolean` later on.

What do you think?

Reply:
Done